### PR TITLE
python3Packages.doorbirdpy: init at 2.1.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 127205` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.doorbirdpy</li>
    <li>python39Packages.doorbirdpy</li>
  </ul>
</details>

File: pkgs/development/python-modules/doorbirdpy/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    pname = "DoorBirdPy";
+    inherit version;
+    sha256 = "ed0e94953cdf96111c7f73c5fcf358f65dc0ff5e47f63fc057bf18ca7512e606";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  # no tests on PyPI, no tags on GitLab
```
Comment (fabaff) [inline pkgs/development/python-modules/doorbirdpy/default.nix:21]: Let's hope that the tests get added to the PyPI release or the source is tagged.

https://gitlab.com/klikini/doorbirdpy/-/issues/3

File: pkgs/development/python-modules/doorbirdpy/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  doCheck = false;
+
+  pythonImportsCheck = [ "doorbirdpy" ];
+
+  meta = with lib; {
+    description = "Python wrapper for the DoorBird LAN API v0.21";
```
Comment (fabaff) [inline pkgs/development/python-modules/doorbirdpy/default.nix:27]: I think that it doesn't make sense to add the API version to the description.

The [documentation](https://www.doorbird.com/downloads/api_lan.pdf) says "Revision: 0.29".

**Review by fabaff — APPROVED** [review]
Looks good to me.

File: pkgs/development/python-modules/doorbirdpy/default.nix (hunk 2 above)
Comment (dotlambda) [reply pkgs/development/python-modules/doorbirdpy/default.nix:27]: The README and setup.py say 0.21, but I'll remove it.

File: pkgs/development/python-modules/doorbirdpy/default.nix (hunk 1 above)
Comment (jonringer) [reply pkgs/development/python-modules/doorbirdpy/default.nix:21]: technically, this just makes it as much of a guarantee as a normal pypi install :)

**Pull request merged**